File: csharp/Controllers/ValuesController.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Threading.Tasks;
using Microsoft.AspNetCore.Mvc;

namespace csharp.Controllers
{
    [Route("/")]
    [ApiController]
    public class ValuesController : ControllerBase
    {
        // GET api/values
        [HttpGet]
        public ActionResult<string> Get()
        {
            return "Hello World!";
        }
    }
}

File: vb/ValuesController.vb
Imports System
Imports Microsoft.AspNetCore.Mvc

    <Route("/")>
    <ApiController>
    Public Class ValuesController
        Inherits ControllerBase

        <HttpGet>
        Public Function [Get]() As ActionResult(Of String)
            Return "Hello World!"
        End Function
    End Class
